### ISSUE TITLE
.btex replacement texture for UI elements elements such as buttons wont work again . 

### ISSUE THREAD

**Issue by faospark** [Kizari/Flagrum#106]
Hey buddy we have encountered this issue before many versions ago and it seems it is back 
no matter what type of texture replacement you do for the ui it seem to be not replaced in game at all 
![Screenshot_279](https://github.com/Kizari/Flagrum/assets/11664809/9f03b03a-0811-4155-ad7a-97a9bc88bbf8)

**Comment by Kizari** [comment]
Oh dear, Flagrum really has it out for you! Funnily enough it's a completely different cause this time. It seems that the mod loader isn't kicking in fast enough to replace the HUD images as they're loaded *really* early in the startup process. Initially I did have it loading earlier, but it caused some really insane problems—we're talking messing up the entire render pipeline, so I moved it a little further along to avoid that. I'll see what can be done, I know there will be a way around this.

**Comment by Kizari** [comment]
This is fixed in Flagrum 1.5.5.

**Comment by Kizari** [comment]
Ack, sorry about that. I wasn't able to test the actual result on my end since I don't get xbox prompts when I play. I did verify that it's loading before any game files before publishing though. Not really sure what more can be done here since it seems to be ignoring the preloading of the mod file despite it being loaded before the actual file it replaces now. Haven't had that problem with any other files, not even the 20,000+ instructions for the Moogle Chocobo Carnival. I can do some research into other potential solutions when my time is freed up a bit, because having to manually replace the file is not ideal.

**Comment by faospark** [comment]
yeah i understand. I knew manually replacing it is not ideal. in my mods page on nexus ive added a work around using special k since it know it has a texture replacement feature . hopefully you find a solution eventually...  take your time bud.

**Comment by junmeimccloud** [comment]
I'm having the same issue. Switch Pro button prompts deom whereswarren's mod don't load. Tested on 1.5.12

**Comment by Kizari** [comment]
Have tested this on the updated mod loader in Flagrum 1.5.17 and it works again now on my end (still using the version of the mod that was current at the time the issue opened, so not sure if the mod listing was updated to compensate). If any issues persist on your end, please leave a comment in #174 with as much detail as possible.

**Comment by faospark** [comment]
> Have tested this on the updated mod loader in Flagrum 1.5.17 and it works again now on my end (still using the version of the mod that was current at the time the issue opened, so not sure if the mod listing was updated to compensate). If any issues persist on your end, please leave a comment in [#174](https://github.com/Kizari/Flagrum/issues/174) with as much detail as possible.

ill check later . been a while :)
